modules/aws_fsx_ontap/main.tf

```diff
@@ -66,10 +66,10 @@ data "aws_iam_policy_document" "worker_fsx_ontap" {
   }
 }
 
-resource "aws_iam_policy" "worker_fsx_ontap" {
+resource "aws_iam_policy" "fsx_ontap" {
   name_prefix = "${var.prefix}-fsx-ontap"
-  description = "EKS worker node fsx_ontap policy for cluster ${var.cluster_name}"
-  policy      = data.aws_iam_policy_document.worker_fsx_ontap.json
+  description = "FSx policy document for user ${data.aws_iam_user.terraform.user_name}"
```

Comment:
This is the actual policy so document should be removed from the description here